arachni/Dockerfile

```diff
@@ -1,22 +1,45 @@
-FROM unit9/base:latest
+FROM unit9/python-uwsgi:latest
 
 MAINTAINER Artiom Vasiliev <artiom@unit9.com>
 
+RUN apt-get update \
+    && apt-get install -y build-essential \
+    && rm -rf /var/cache/apt /var/lib/apt/lists/*
+
+# Install redis and arachni
 RUN cd /tmp \
-    && curl -LO https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 \
-    && tar xjf phantomjs-2.1.1-linux-x86_64.tar.bz2 \
-    && mv phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/local/bin/ \
-    && rm -r phantomjs-2.1.1-linux-x86_64 \
+    && curl -LO http://download.redis.io/redis-stable.tar.gz \
```

Comment:
It would be better to (in order of most preferred options):

- run redis in a separate container
- install redis from distro's packages